string name
float32 mx
float32 my
float32 a
float32 b
float32 time
float32 k
---